### PR TITLE
Documentation: formatting changes in Documentation #4357

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Fix Formatting Inconsistency in Documentation
------------------
This PR aims to fix the errors in formatting & while building up the documentation locally for `ReadTheDocs` (also tries to fix major build warning in the configuration for **Sphinx**)

### CODE REVIEW

File: doc/source/man/daemons.rst [hunk 1]
```diff
@@ -6,14 +6,11 @@ Daemons CLIs
     :maxdepth: 1
 
     rucio-abacus-account
-    rucio-abacus-collection-replica
```
Comment (bari12) [inline doc/source/man/daemons.rst]: This exists, instead of removing, please add the man-link :-)

File: doc/source/man/daemons.rst [hunk 2]
```diff
@@ -6,14 +6,11 @@ Daemons CLIs
...
     rucio-abacus-rse
-    rucio-auditor
```
Comment (bari12) [inline doc/source/man/daemons.rst]: Still exists

File: doc/source/man/daemons.rst [hunk 3]
```diff
@@ -6,14 +6,11 @@ Daemons CLIs
...
     rucio-atropos
     rucio-automatix
     rucio-bb8
     rucio-c3po
-    rucio-cache-client
```
Comment (bari12) [inline doc/source/man/daemons.rst]: exists

File: doc/source/man/daemons.rst [hunk 4]
```diff
@@ -34,6 +30,5 @@ Daemons CLIs
     rucio-minos-temporary-expiration
     rucio-necromancer
     rucio-reaper
-    rucio-replica-recoverer
```
Comment (bari12) [inline doc/source/man/daemons.rst:37]: exists

**Review by bari12 — CHANGES_REQUESTED** [review]

File: doc/source/man/daemons.rst (hunk 1 above)
Comment (DhruvSondhi) [reply doc/source/man/daemons.rst]: Sorry, @bari12, I am not able to find these pages [here](https://github.com/rucio/rucio/tree/master/doc/source/man) [man directory of the `doc/source`]... Neither is this present anywhere else except in the `bin` directory of Rucio ... Can you please point me in the correct direction for the same?

Comment (bari12) [reply doc/source/man/daemons.rst]: Yes, I meant the pages should be added. :-) The page itself is just a simple reference to parse the info from the bin/ anyway.

Comment (DhruvSondhi) [reply doc/source/man/daemons.rst]: Sure ... I will add them :)

**Review by bari12 — APPROVED** [review]

**Pull request merged**